tools/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -176,62 +167,62 @@ abstract class AppBase(
     }
   }
 
-  protected def checkJdbcScan(sqlID: Long, planInfo: SparkPlanInfo): Unit = {
-    val allJdbcScan = getJdbcInPlan(planInfo)
-    if (allJdbcScan.nonEmpty) {
-      dataSourceInfo += DataSourceCase(sqlID, "JDBC", "unknown", "unknown", "")
+  // This tries to get just the field specified by tag in a string that
+  // may contain multiple fields.  It looks for a comma to delimit fields.
+  private def getFieldWithoutTag(str: String, tag: String): String = {
+    val index = str.indexOf(tag)
+    // remove the tag from the final string retruned
+    val subStr = str.substring(index + tag.size)
+    val endIndex = subStr.indexOf(", ")
+    if (endIndex != -1) {
+      subStr.substring(0, endIndex)
+    } else {
+      subStr
     }
   }
 
   // This will find scans for DataSource V2, if the schema is very large it
   // will likely be incomplete and have ... at the end.
   protected def checkGraphNodeForBatchScan(sqlID: Long, node: SparkPlanGraphNode): Unit = {
```

Comment:
This method now checks more than just BatchScan. I am not sure if the method name needs to be changed.

Reply:
I don't think its a big deal but I'll update to Reads